sdks/python/apache_beam/testing/test_stream.py

```diff
@@ -171,13 +180,14 @@ class TestStream(PTransform):
   time. After all of the specified elements are emitted, ceases to produce
   output.
   """
-  def __init__(self, coder=coders.FastPrimitivesCoder(), events=None):
+  def __init__(
+      self, coder=coders.FastPrimitivesCoder(), events=None, output_tags=None):
     super(TestStream, self).__init__()
     assert coder is not None
     self.coder = coder
     self.watermarks = {None: timestamp.MIN_TIMESTAMP}
     self._events = [] if events is None else list(events)
```

Comment:
assert that events tags is a subset of output_tags

Reply:
Done.